src/main/java/build/buf/protovalidate/CustomOverload.java

```diff
@@ -20,15 +20,14 @@
 import com.google.common.primitives.Bytes;
 import inet.ipaddr.IPAddress;
 import inet.ipaddr.IPAddressString;
-import jakarta.mail.internet.AddressException;
```

Comment:
Are we now able to remove this dependency as well?

Reply:
mos def. removed.